qiskit/transpiler/preset_passmanagers/generate_preset_pass_manager.py

```diff
@@ -226,13 +228,24 @@ def generate_preset_pass_manager(
                 raise ValueError(
                     f"Gates with 3 or more qubits ({gate}) in `basis_gates` or `backend` are "
                     "incompatible with a custom `coupling_map`. To include 3-qubit or larger "
-                    " gates in the transpilation basis, use a custom `target` instance instead."
+                    " gates in the transpilation basis, provide a custom `target` instead."
                 )
 
     if target is None:
         if backend is not None and _no_loose_constraints:
             # If a backend is specified without loose constraints, use its target directly.
             target = backend.target
+        elif _adjust_dt:
+            # If a backend is specified with loose dt, use its target and adjust the dt value.
+            target = copy.deepcopy(backend.target)
+            target.dt = dt
+            # we must update the instruction properties to internally clear the value of
+            # _instruction_durations in the target so that the new dt is saved
+            inst_name = list(target.operation_names)[0]
+            props_map = target[inst_name]
+            for qargs, props in props_map.items():
+                target.update_instruction_properties(inst_name, qargs, props)
+                break
```

Comment:
I was thinking I'd probably write this as:

```python
qargs, props = next(iter(props_maps.items()
target.update_instruction_properties(inst_name, qargs, props)
```

because the for loop looks a little weird with a unconditional break. But actually what you have here is better because it tolerates an empty properties map correctly.


But writing all of that I'm thinking this actually isn't robust against ideal gates in a target where things are none. Like what happens if we get unlucky and the first operation in the list is an `IfElseOp` or something else that's ideal. I think we might want to change the logic here to iterate until we either successfully update the instruction properties with something or iterate over all the elements.

That being said I know you did this to avoid using a private interface in the target. But I think it would be totally fine in this case to just do and simplify this:

```suggestion
            target._instruction_durations = None
```
just view it as `pub(crate)` usage

Reply:
I am on board with the suggestion if you are :)